msteams-platform/whats-new.md

```diff
@@ -17,6 +17,7 @@ Discover Microsoft Teams platform features that are generally available (GA) and
 
 | Date | Update | Find here  |
 | --- | --- | --- |
+|12/28/2021| Introduced step-by-step guide to do Teams file upload bot | Build bots > Send and receive files > [step-by-step guide to grant Teams file upload bot](sbs-file-handling-in-bot.yml) |
```

Comment:
Global change in the Whats new page: Replace "to do file upload bot" and "to grant Teams file upload bot" with how to upload files to Teams from a bot

Reply:
made changes

Reply:
changes made